lib/kerror/codes/2-api.json

```
@@ -128,6 +128,12 @@
           "code": 9,
           "message": "Admin user is already set.",
           "class": "PreconditionError"
+        },
+        "too_many_login_requests": {
+          "description": "The request has been refused because logins attempts per second has been exceeded for this user",
+          "code": 10,
+          "message": "Rejected: requests for logins per second exceeded for this user.",
+          "class": "TooManyLoginRequestsError"
```

This property is used to construct the error and must be the name of one of our error classes.  
Those errors are defined in [another repository](https://github.com/kuzzleio/kuzzle-common-objects/tree/master/lib/errors).  
You can use the `TooManyRequestsError` class for example:
```suggestion
          "class": "TooManyRequestsError"
```